YAFC/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -298,22 +298,22 @@ private class EntityColumn : ProductionTableDataColumn {
 
             private void ShowAccumulatorDropdown(ImGui gui, RecipeRow recipe, Entity accumulator) {
                 gui.ShowDropDown(imGui => {
-                    imGui.BuildInlineObjectListAndButton<EntityAccumulator>(Database.allAccumulators, DataUtils.DefaultOrdering,
```

Comment:
The type was removed there. Could you please elaborate on that?

Reply:
Not really... :smile:
I use(d) the formatting tools to rename variables/functions, so I didn't notice.
Let me check, maybe it became/is obsolete and by refactoring that line other tools removed it automatically? :thinking:

Reply:
Yes, it was indeed something that got fixed by accident: [Simplify name (IDE0001)](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0001?pivots=lang-csharp-vb)

I'll revert to keep this PR clean (there are lots of other fixes pointed out by editorconfig to be made :stuck_out_tongue: )

Reply:
I use the 'rename' functionality to fix the names of the variables/functions. This one got changed as a 'dependency'. I guess it triggered the 'full cleanup' on that line or something?

Reply:
As far as I know there is no global clean up. For this PR I just opened all code/MD files and looked if there were cSpell warnings to fix.
![image](https://github.com/have-fun-was-taken/yafc-ce/assets/171827/ee1c2918-9920-48f9-9e5c-1a37b040cc76)

Reply:
Regarding the revert that you did, the rest of that function also got changed in the fix, but it was not reverted.

Reply:
Oh... I didn't notice :disappointed:

Reply:
FYI After reverting, I used 'rename' to rename `accum` -> `accumulator` again and it triggered the other cleanups. 
As a matter of fact it was impossible to use the rename without triggering and I changed both manually.

So it looks at surrounding code and triggers the cleanup? :thinking:

Reply:
Interesting. I'm not against that change, just wanted to know why exactly that function was changed beyond the rename.

Reply:
Sure, if you feel like doing that, I'll be happy to help with the review. But this is optional.

Reply:
Then let's leave it :smile: 
I don't feel too confident what is a right change and what is not. For the grammar/typos/US vs UK EN I got triggered. The other cleanups a whole lot less.